nemesis/arena_prep.py

```diff
@@ -3081,16 +3178,32 @@ def gagazet(cap_num: int = 10):
                 checkpoint = 12
 
             # NEA decisions
-            if neArmor == True and checkpoint in [7, 19, 23, 33]:
+            if neArmor and checkpoint in [7, 19, 23, 33]:
```

Comment:
How do we still have camelCase? Has shen not gotten in here yet?

Reply:
I've updated the camel case ones that I could find. We'll catch the rest later.